benchmarks/tpcds/schemas/psql_native_partitioned_id_hash/indexes.sql

```diff
@@ -0,0 +1,167 @@
+--create index idx_customer_demographics_1      on customer_demographics (cd_marital_status,        cd_education_status);
```

Comment:
Remove the comments?

Reply:
Are the same comments as in any other index file in all TPCDS schema, I prefer to leave them there so all index.sql files are as similar as possible.